src/plugins/auto/auto_schedule.hpp

```diff
@@ -18,6 +18,8 @@ struct AutoLoadContext {
     std::atomic<bool> isEnabled = {false};
     std::atomic<bool> isAlready = {false};
     std::atomic<bool> isLoadSuccess = {false};
+    std::atomic<bool> isReloadSuccess = {false};
```

Comment:
do we really need this flag？ can't we use isLoadSuccess for loadContext[FALLBACKDEVICE]

Reply:
This flag to show if load fallback device ready, isLoadSuccess is set before GenerateWorkers, so add this flag to show the fallback device is ready after GenerateWorkers, just like isAlready flag. but if we use isAlready flag, if the infer failed again in fallback device, it can't reset isAlready to false because there maybe other infer still work in the pipeline, these infer request will use isAlready  flag in ScheduleToWorkerInferRequest(). so must keep isAlready always ok if use fallback device.